app/src/org/commcare/dalvik/activities/CommCareSetupActivity.java

```diff
@@ -140,7 +146,7 @@
     public void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         CommCareSetupActivity oldActivity = (CommCareSetupActivity)this.getDestroyedActivityState();
-        this.mFromManager = this.getIntent().
+        this.fromManager = this.getIntent().

```

Comment:
I don't have super strong preferences here, although I do like the
mNotation more than most types because more than just clarifying meaning in
a class, it also makes it hard to, for instance, confuse a variable
assignment with a class member assignment (and I really dislike seeing
"this" everywhere). We don't do it super consistently because we started
not-doing-it and drifted to doing it more often.

-Clayton

On Mon, Aug 17, 2015 at 10:53 AM, Phillip Mates <notifications@github.com>
wrote:

> In app/src/org/commcare/dalvik/activities/CommCareSetupActivity.java
> <https://github.com/dimagi/commcare-odk/pull/523#discussion_r37195711>:
>
> > @@ -140,7 +146,7 @@
> >      public void onCreate(Bundle savedInstanceState) {
> >          super.onCreate(savedInstanceState);
> >          CommCareSetupActivity oldActivity = (CommCareSetupActivity)this.getDestroyedActivityState();
> > -        this.mFromManager = this.getIntent().
> > +        this.fromManager = this.getIntent().
>
> I wasn't aware that was a style convention we followed.
> My reasoning against the 'm' is that modern IDEs highlight class fields,
> so there isn't a reason to also label them with an 'm'. Plus we don't seem
> very good at making sure that variables prefixed by 'm' are actually class
> fields...
>
> Not prefixing with 'm' does go against Google's style guide for Android
> platform dev, but I'm fine with that, since we don't work in that codebase.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dimagi/commcare-odk/pull/523/files#r37195711>.
>